packages/manager/modules/iam/src/iam.routing.js

```diff
@@ -0,0 +1,131 @@
+import FeatureAvailabilityResult from '@ovh-ux/ng-ovh-feature-flipping/src/feature-availability-result.class';
+
+import {
+  ALERT_ID,
+  FEATURE,
+  GUIDE,
+  UNAVAILABLE_STATE_NAME,
+} from './iam.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('iam', {
+    url: '/iam',
+    template: `
+      <div class="iam">
+        <div data-ovh-alert="${ALERT_ID}" class="mt-3"></div>
+        <div data-ui-view></div>
+      </div>
```

Comment:
**nitpick:** Can be moved to HTML file to have better readability and in case if the template can grow in future.